share/picongpu/examples/FieldAbsorberTest/cmakeFlags

```diff
@@ -30,62 +30,66 @@
 #   - increase by 1, no gaps
 
 # Test that exponential damping compiles in 3D and 2D
-flags[0]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_ABSORBER_SIZE=8'"
-flags[1]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_ABSORBER_SIZE=8;-DPARAM_DIMENSION=DIM2'"
+flags[0]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee< CurrentInterpolation >;-DPARAM_ABSORBER_SIZE=8'"
+flags[1]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee< CurrentInterpolation >;-DPARAM_ABSORBER_SIZE=8;-DPARAM_DIMENSION=DIM2'"
+# Test that arbitrary-order solver compiles in 3D and 2D
+flags[2]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=ArbitraryOrderFDTD< 4 MY_AOFDTD_COMMA CurrentInterpolation >;-DPARAM_ABSORBER_SIZE=8'"
+flags[3]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=ArbitraryOrderFDTD< 4 MY_AOFDTD_COMMA CurrentInterpolation >;-DPARAM_ABSORBER_SIZE=8;-DPARAM_DIMENSION=DIM2'"
 # Test that PML compiles in 3D and 2D
-flags[2]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2'"
-flags[3]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+flags[4]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2'"
+flags[5]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+
 
 # The following tests are for absorber performance and demonstration of
 # reasonable parameters, commented out to make compile-time tests faster
 
 # Exponential damping in 3D, default strength
-#flags[4]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_ABSORBER_SIZE=12'"
-#flags[5]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_ABSORBER_SIZE=16'"
-#flags[6]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_ABSORBER_SIZE=24'"
-#flags[7]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_ABSORBER_SIZE=32'"
+#flags[6]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_ABSORBER_SIZE=12'"
```

Comment:
In these commented test the `< CurrentInterpolation >` is missing:
```suggestion
#flags[6]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee< CurrentInterpolation >;-DPARAM_ABSORBER_SIZE=12'"
```

Reply:
:facepalm: